app/views/campaign_pages/_form.slim

```diff
@@ -1,21 +1,24 @@
 = form_for(@campaign_page, class: 'form-horizontal') do |f|
```

Comment:
Ah, sorry, I'd missed that you did a separate edit form somehow. Removing this comment.